roles/ovirt-clean-engine/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+- block:
+    - name: Obtain SSO token
+      ovirt_auth:
+          url: "{{ DR_sites[target_host].url }}"
+          username: "{{ DR_sites[target_host].username }}"
+          # TODO: Set password per site
+          password: "{{ password }}"
+          ca_file: "{{ DR_sites[target_host].ca_file }}"
+
+    # TODO: execute OVF_UPDATE
+    # Get all the running VMs and shut them down
+    - name: Fetch VMs in the setup
+      ovirt_vms_facts:
+          pattern:  status != down
+          auth: "{{ ovirt_auth }}"
+
+    - include_tasks: tasks/shutdown_vms.yml vms={{ item }}
+      with_items: "{{ ovirt_vms }}"
+
+    # TODO: add a test of ISO and export storage domain.
```

Comment:
I've changed this TODO, we can't support export storage domain today using ovirt-ansible since export storage domain can not be attached with dirty metadata, I will open a separate bug on it